Jenkinsfile

```diff
@@ -31,18 +31,39 @@ pipeline{
           }
         }
       }
-      stage('Deploy'){}
+      stage('Deploy'){
+        deployToS3('./bin/**')
```

Comment:
Because we're using a custom defined script, I believe you will need to wrap this with a `script` block. So it would look like: `script { deployToS3('./bin/**') }`